docs/how-write-documentation.md

```diff
@@ -17,6 +17,45 @@ IntelliJ:
 * Editor font: Fira Code, 16
 * Enable ligatures: true
 
+# GIF
+
+## Preparation
+
+Use [simplescreenrecorder](http://www.maartenbaert.be/simplescreenrecorder/) to record .mkv
+Use [giflossy](https://github.com/pornel/giflossy/releases/tag/lossy/1.82.1) for GIF compression (aliased as gifsicle)
+
+## Recording MKV
+
+Configure simplescreenrecorder:
+
+* Set frame rate to 10
+* Don't record audio
+* Record a fixed rectangle at 1280x960 at (0,0)
+
+Resize and move window to upper-left corner:
+
+```
+xdotool search --name "graph-examples" windowsize 1280 960 windowmove 0 0
+```
+
+Save recorded video with screencast.mkv name
+
+## Generate GIF
+
+cd into directory where screencast.mkv resides
+
+Execute following commands
+
+```sh
+mkdir frames
```

Comment:
can you put this in a function that accepts name for a gif
And also do all operations in `/tmp` folder?